owncloudApp/src/androidTest/java/com/owncloud/android/shares/data/OCShareDaoTest.kt

```diff
@@ -284,15 +284,51 @@ class OCShareDaoTest {
         assertEquals("userName", text2Shares[0].shareWith)
     }
 
+    @Test
+    fun updatePrivateShare() {
+        ocShareDao.insert(createDefaultPrivateShare())
+
+        ocShareDao.update(
+            createDefaultPrivateShare(permissions = 17)
+        )
+
+        val textShares = getValue(
+            ocShareDao.getSharesAsLiveData(
+                "/Texts/text1.txt", "admin@server", listOf(ShareType.USER.value)
```

Comment:
This is a bit difficult to read. How should I know that "/Texts/text1.txt", "admin@server", etc... correspond to the fields of the share returned by 'createDefaultPrivasteShare()' ?

I would save the value of createDefaultPrivateShare() in a variable at the beginning, then use the fields of that variable here to pass file, user, etc... That way it's clear that we are getting the same share that we inserted first.

Reply:
Done